modules/gke-cluster/variables.tf

```diff
@@ -74,6 +74,8 @@ variable "dashboard_disabled" {
   description = "Disable Kubernetes Dashboard"
 }
 
+# Setting both password and username to empty strings (module default)
```

Comment:
I think this will be easier for a user to notice if it's in the Description.

Reply:
How would you prefer this? In username? Password? Both?

The behavior here mirrors the one of TF's `container_cluster` and the one of GCP api & `gcloud` tool, not sure we should do much more explanation here. See the "basic auth" section at https://cloud.google.com/sdk/gcloud/reference/container/clusters/create - the behavior is quite complex.

Reply:
I was thinking that you keep what you have as a comment, but in the description for each variable.

I agree that it's not our job to exhaustively document the gcloud API here :). I think a hint is warranted since I can maybe guess the default behavior but it is not trivially clear to me. Contrast this case with `zone` (pretty clear what it's for; no comment needed) or `addons` (pretty clear that I need to look at the API docs if I want to know more).

Move it to description, leave it as-is, or remove it -- I leave it up to you.

Reply:
thanks for the clarification, I'll move it to descriptions of both username & password